### ISSUE TITLE
Python 3.10 incompatibility. Solution - remove requirement of numpy<v1.20.0

### ISSUE THREAD

**Issue by Martin-Laclaustra** [r9y9/nnmnkwii#122]
TL;DR;
https://github.com/r9y9/nnmnkwii/blob/86cec7705d4598f6ce561f002b36a5893b114e6c/pyproject.toml#L6
should be changed to just `"numpy",`
thanks to a recent update in https://github.com/r9y9/pysptk that allows it.
That solves incompatibility with Python 3.10.

### Long explanation
This module suffered the same problem as pysptk: https://github.com/r9y9/pysptk/issues/91
It was requesting a former version of numpy (19), which happens to be incompatible with Python 3.10. Thus, neither of these modules could be installed.
The requirement in this module was forced by its dependency on pysptk.
Aug, 7th commit released the requirement from pysptk, https://github.com/r9y9/pysptk/commit/93896a93074cf5483b56c7e1daf4948708749325.
Thus, this module can also be fixed now.

Thank you for all your generously shared work!
